### PR TITLE
[DQM/L1TEMU] Removing the condition for data/emul sizes being equal

### PR DESCRIPTION
**Pull request by vukasinmilosevic**
Removing the condition for data/emul sizes being equal in order to fill problematic sums histograms.

### CODE REVIEW

**Comment by thomreis** [comment]
Hi @vukasinmilosevic 
why is this happening even if the comment above the changed line says it should never happen?

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
please test

File: DQM/L1TMonitor/src/L1TdeStage2CaloLayer2.cc [hunk 1]
```diff
@@ -1012,568 +1012,571 @@ bool L1TdeStage2CaloLayer2::compareSums(
   double dataPhi = 0;
   double emulPhi = 0;
 
-  l1t::EtSumBxCollection::const_iterator dataIt = dataCol->begin(currBx);
+  l1t::EtSumBxCollection::const_iterator dataIt;
   l1t::EtSumBxCollection::const_iterator emulIt = emulCol->begin(currBx);
 
   // if either data or emulator collections are empty, or they have different
   // size, mark the event as bad (this should never occur in normal running)
```
Comment (thomreis) [inline DQM/L1TMonitor/src/L1TdeStage2CaloLayer2.cc]: Hi @vukasinmilosevic 
can you please also update the comment corresponding to the changed code?

Comment (vukasinmilosevic) [reply DQM/L1TMonitor/src/L1TdeStage2CaloLayer2.cc]: Will do it now.

**Comment by thomreis** [comment]
Can you also check the indentation please. Better not use tabs.

**Comment by vukasinmilosevic** [comment]
Hello @andrius-k, could you wait with merging this PR (as well as the backport)? The dqm playback testing is returning a different result than my local testing.

**Comment by andrius-k** [comment]
Hi @vukasinmilosevic, sure, it can wait.

**Comment by andrius-k** [comment]
Hi @vukasinmilosevic, the indentation of your recent changes seems very inconsistent. Could you please take a look at it?

**Comment by thomreis** [comment]
Hi @vukasinmilosevic 
there are still some lines with inconsistent indentation. Please check the diff in github to find them.

**Comment by thomreis** [comment]
please test

**Comment by andrius-k** [comment]
+1

**Comment by vukasinmilosevic** [comment]
Hi @fabiocos, by looking at the log for 136.731 all the histograms that are failing the BinToBin comparison at the end are the DQM plots that are affected by this PR.

**Comment by fabiocos** [comment]
+1

**Pull request merged**